src/app/Plans/PlansPage.tsx

```diff
@@ -75,13 +73,11 @@ const PlansPage: React.FunctionComponent = () => {
                     content={addPlanDisabledObj.content}
                   >
                     <div className={`${spacing.mtMd}`}>
-                      <Button
-                        isDisabled={addPlanDisabledObj.isTooltipEnabled}
-                        onClick={toggleWizard}
-                        variant="primary"
-                      >
-                        Create migration plan
-                      </Button>
+                      <Link to="/planwizard">
```

Comment:
Let's see if we can keep the URL paths roughly based on the breadcrumb bar, so this should be a route under `/plans`. Maybe `/plans/create`?

Reply:
Also does this link properly disable with the button is disabled? Not sure how that works when it's wrapped around it like this. If not we may need to either only render the <Link> around it when the button would be enabled, or handle the link behavior as an onClick instead.

Reply:
That's a good point. I'll done a quick unary fix before adding a wrapper for long run.